frontend/components/auction/debt/DebtAuctionBidTransactionTable.vue

```diff
@@ -0,0 +1,161 @@
+            <div>
+                <FormatCurrency v-if="auction.combinedBidFeesDai" :value="auction.combinedBidFeesDai" currency="DAI" />
+                <span v-else class="opacity-50">Unknown</span>
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>Price on Uniswap</div>
+            <div>
+                <format-currency
+                    v-if="auction.marketUnitPrice && isActive"
+                    :value="auction.marketUnitPrice"
+                    :decimal-places="6"
+                    currency="DAI"
+                />
+                <span v-else class="opacity-50">Unknown</span> per MKR
+            </div>
+        </div>
+        <div class="flex justify-between font-bold">
+            <div>Price after the bid</div>
+            <div>
+                <format-currency
+                    v-if="unitPriceAfterBid && isActive && !isBidAmountNaN"
+                    :value="unitPriceAfterBid"
+                    :decimal-places="6"
+                    currency="DAI"
+                />
+                <span v-else class="opacity-50">Unknown</span> per MKR
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import type { DebtAuctionTransaction } from 'auctions-core/src/types';
+import Vue from 'vue';
+import BigNumber from 'bignumber.js';
+import BidInput from '../../common/inputs/BidInput.vue';
+import TimeTill from '~/components/common/formatters/TimeTill.vue';
+import FormatCurrency from '~/components/common/formatters/FormatCurrency.vue';
+
+export default Vue.extend({
+    components: {
+        TimeTill,
+        FormatCurrency,
+        BidInput,
+    },
+    props: {
+        auction: {
+            type: Object as Vue.PropType<DebtAuctionTransaction>,
+            required: true,
+        },
+    },
+    data() {
+        return {
+            inputBidAmount: undefined as BigNumber | undefined,
+        };
+    },
+    computed: {
+        isActive(): boolean {
+            return this.auction.state === 'just-started' || this.auction.state === 'have-bids';
+        },
+        isBidAmountNaN(): boolean {
+            return !!this.inputBidAmount?.isNaN();
+        },
+        unitPriceAfterBid(): BigNumber | undefined {
+            if (!this.isActive) {
+                return undefined;
+            }
+            if (!this.inputBidAmount) {
+                return this.auction.nextMaximumLotReceived.dividedBy(this.auction.receiveAmountMKR || 1);
+            }
+            return this.inputBidAmount.dividedBy(this.auction.receiveAmountMKR || 1);
+        },
+    },
+    watch: {
+        inputBidAmount: {
+            immediate: true,
+            handler(inputBidAmount) {
+                this.$emit('inputBidAmount', inputBidAmount);
+            },
+        },
+    },
+    methods: {
+        setInputBidAmount(value: BigNumber | undefined) {
+            if (this.isActive) {
+                this.inputBidAmount = value;
```

Comment:
Technically it's not bid amount, it's `inputCompensationAmount` or if we use already existing terminology:
```suggestion
                this.receiveAmountMKR = value;
```
But then many other `bid`-related name should be changed, like `HighestDebtBidCheckPanel` to `DebtLatestBidCheckPanel`